docs/2.0/05-filters-en.md

```diff
@@ -37,6 +38,20 @@ match-dml-op = "delete"
 
 For the above configuration, all the `delete` `DML` message with `schema` set to `test` are rejected.
 
+## `accept` configuration
+
+The `reject` Filter rejects specific Input messages. Then these messages will not be the next Filter or Output.
```

Comment:
The `accept` is a whitelist filter that accepts...

Reply:
done